scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -32,14 +32,14 @@ RUN apt-get update; \
 		software-properties-common \
 		ninja-build git wget \
 		libbz2-dev zlib1g-dev git curl uuid-dev \
-		pkg-config openjdk-8-jdk liblzma-dev unzip mlton m4; \
+		pkg-config openjdk-8-jdk liblzma-dev unzip mlton m4 jq; \
     apt-get install -qy python3-pip;
 
-# Install cmake 3.21.2 (minimum requirement is cmake 3.10)
-RUN wget https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2-Linux-x86_64.sh; \
-    test "$(sha256sum cmake-3.21.2-Linux-x86_64.sh)" = "3310362c6fe4d4b2dc00823835f3d4a7171bbd73deb7d059738494761f1c908c  cmake-3.21.2-Linux-x86_64.sh"; \
-    chmod +x cmake-3.21.2-Linux-x86_64.sh; \
-    ./cmake-3.21.2-Linux-x86_64.sh --skip-license --prefix="/usr"
+# Install cmake 3.22.2 (minimum requirement is cmake 3.10)
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.22.2/cmake-3.22.2-linux-x86_64.sh; \
+    test "$(sha256sum cmake-3.22.2-linux-x86_64.sh)" = "faaa0465b72fe467fa193954c9bb67141048a2c7acc3236975ac20fdf241582e  cmake-3.22.2-linux-x86_64.sh"; \
+    chmod +x cmake-3.22.2-linux-x86_64.sh; \
+    ./cmake-3.22.2-linux-x86_64.sh --skip-license --prefix="/usr"
```

Comment:
For `ubuntu1604.clang.ossfuzz` there is already a cmake installed in `/usr/local/bin/cmake` v3.21.1, where `PATH` is set to `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/out` - so the "old" version has higher priorities. Maybe we could just change the `prefix` to install the new version in `/usr/local`.

Reply:
I will now change cmake installation `prefix` to `/usr/local`.